executor/ddl.go

```diff
@@ -250,8 +250,11 @@ func (e *DDLExec) executeDropTable(s *ast.DropTableStmt) error {
 			return errors.Trace(err)
 		}
 	}
-	if len(notExistTables) > 0 && !s.IfExists {
-		return infoschema.ErrTableDropExists.GenWithStackByArgs(strings.Join(notExistTables, ","))
+	if len(notExistTables) > 0 {
+		if !s.IfExists {
+			return infoschema.ErrTableDropExists.GenWithStackByArgs(strings.Join(notExistTables, ","))
+		}
+		e.ctx.GetSessionVars().StmtCtx.AppendNote(infoschema.ErrTableDropExists.GenWithStackByArgs(strings.Join(notExistTables, ",")))
```

Comment:
use a loop to AppendNote for every not exists tables.

Reply:
Ok, it has been changed.